src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,102 @@
+import { useSession, signIn, signOut } from "next-auth/react";
+import Link from 'next/link';
+import styles from './Menu.module.css';
+import Button from "@/src/components/Button/Button";
+import { useState, useEffect } from "react";
+
+const Menu = (props:any) => {
+    const {data: session} = useSession();
+    const [screenWidth, setScreenWidth] = useState(0);
+
+    useEffect(() => {
+        setScreenWidth(window.innerWidth);
+        const handleResize = () => setScreenWidth(window.innerWidth);
+        window.addEventListener('resize', handleResize);
+        return () => {
+          window.removeEventListener('resize', handleResize);
+          if(screenWidth >= 640){
+            document.body.style['overflow'] = 'auto';
+            props.setOpen(!open);
```

Comment:
czy tutaj jest dostepne open? wydaje mi sie ze tylko props.open

Reply:
w sumie jezeli ekran jest wiekszy niz mobile to mozesz zawsze zamykac mobilne menu po prostu, czyli setOpen(false) i elo

Reply:
> w sumie jezeli ekran jest wiekszy niz mobile to mozesz zawsze zamykac mobilne menu po prostu, czyli setOpen(false) i elo

zmieniłem w najnowszym commicie na false